metadata.json

```diff
@@ -3,10 +3,12 @@
   "category": "office",
   "icon": "icon.png",
   "title": {
-    "en_EN": "Writer"
+    "en_EN": "Writer",
+    "pt_BR": "Writer"
```

Comment:
There isn't a native word for "Writer" or something similar that could be used ? :)

Reply:
I didn't change it because it's the name of the app, so I didn't think it was necessary to change it. Do you want it to change? It will be similar to textpad

Reply:
You bring up a pretty valid point there, but I think it makes sense to change the name as well :thinking: At least if there's a title that closely resembles the english one.

Reply:
Right.
I have an idea. I'm going to change :)